composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "catharsisjelly/workflows",
```

Comment:
@jdfwarrior if you accept this change then we need to change this to your repo instead :)